cabot/cabotapp/models.py

```diff
@@ -441,6 +442,12 @@ class StatusCheck(PolymorphicModel):
         help_text='Number of successive failures permitted before check will be marked as failed. '
                   'Default is 0, i.e. fail on first failure.'
     )
+    run_delay = models.PositiveIntegerField(
+        default=0,
+        validators=[MinValueValidator(0)],
```

Comment:
Is the custom validator necessary when using `PositiveIntegerField`?

Reply:
You're right!  I copied from another place in which I had used it to ensure all values were >= 1.  Not needed for >= 0.  Removed.